ckanext/datajson/harvester_datajson.py

```diff
@@ -19,20 +20,25 @@ def info(self):
         }
 
     def load_remote_catalog(self, harvest_job):
+        log.info('load_remote_catalog from: {}'.format(harvest_job.source.url))
         req = urllib2.Request(harvest_job.source.url)
         # todo: into config and across harvester
         req.add_header('User-agent', 'Data.gov/2.0')
         try:
             datasets = json.load(urllib2.urlopen(req))
         except UnicodeDecodeError:
             # try different encode
+            log.error('unicode error at {}'.format(harvest_job.source.url))
             try:
+                log.info('trying cp1252')
                 datasets = json.load(urllib2.urlopen(req), 'cp1252')
             except:
+                log.error('trying iso-8859-1')
                 datasets = json.load(urllib2.urlopen(req), 'iso-8859-1')
-        except:
+        except Exception, e:
             # remove BOM
-            datasets = json.loads(lstrip_bom(urllib2.urlopen(req).read()))
+            log.error('trying to remove BOM. Error: {}.'.format(e))
+            datasets = json.loads(lstrip_bom(urllib2.urlopen(req, context=ssl._create_unverified_context()).read()))
```

Comment:
Why are we creating an unverified SSL context?

Reply:
I copied this from another fork. I think someone was trying to avoid errors in any possible way.
If you prefer we can remove this particular exception management @adborden